flutter_news_example/packages/deep_link_client/firebase_deep_link_client/lib/src/firebase_deep_link_client.dart

```diff
@@ -0,0 +1,26 @@
+import 'dart:async';
+
+import 'package:deep_link_client/deep_link_client.dart';
+import 'package:firebase_dynamic_links/firebase_dynamic_links.dart';
+
+/// {@template firebase_deep_link_client}
+/// A FirebaseDynamicLinks implementation of [DeepLinkService].
```

Comment:
typo: `A FirebaseDynamicLinks implementation of [DeepLinkService].` > `A FirebaseDynamicLinks implementation of [DeepLinkClient].`